### PR TITLE
upgrade go version on gitpod

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
the workspace in gitpod is broken because Go is not up to date, so we are not able to build Prometheus on gitpod. This PR is fixing this issue.

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -1,7 +1,18 @@
 FROM gitpod/workspace-full
 
+
+FROM gitpod/workspace-full
```
Comment (juliusv) [inline .gitpod.Dockerfile:4]: This is the same as line 1. Is it meant to be duplicated?

**Comment by juliusv** [comment]
:+1: besides my comment

File: .gitpod.Dockerfile (hunk 1 above)
Comment (Nexucis) [reply .gitpod.Dockerfile:4]: ah no, it's totally a mistake (bad copy and past from Perses 😅. I will remove it.

**Pull request merged**